api/versions.go

```diff
@@ -549,6 +601,87 @@ func convertStats(cont *info.ContainerInfo) []v2.ContainerStats {
 	return stats
 }
 
+func convertStatsToV2(cont *info.ContainerInfo) []v2.ContainerStats {
```

Comment:
Can we separate request handling from conversion? (At least separate files). E.g. something like https://github.com/google/cadvisor/pull/921/files#diff-aac76d5d60dea772214f3f48e0b21705 (might make more sense in a separate PR).

Longer term, I think we should switch to a model closer to the k8s API, with an unversioned internal model & versioned client APIs.